tests/Hedgehog.Linq.Tests/Hedgehog.Linq.Tests.csproj

```diff
@@ -15,5 +15,8 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\Hedgehog\Hedgehog.fsproj" />
+    <Reference Include="Hedgehog">
+      <HintPath>..\..\src\Hedgehog\bin\Debug\netstandard2.0\Hedgehog.dll</HintPath>
```

Comment:
Would it be possible to add a comment here so that others can see the reason we're adding a direct DLL reference? 👀

We can merge this as-is but the comment can help everyone else who hasn't paid attention to this PR.

Reply:
I can just remove this, it's a hack to get a specific tool to work. Thinking about it, this doesn't need to be part of the mainline.

Reply:
It's up to you. Unless @TysonMN has a strong opinion on this.

Reply:
I am all for improving the experience in various IDEs (especially an IDE of our most active contributor!), but this doesn't feel like the right solution to me.  If you still want to pursue this, then I suggest creating a new issue and providing more details about the problem.